src/components/Post.jsx

```diff
@@ -315,130 +338,148 @@ function Post(prop) {
         <div className="post__text">
           {caption && postHasImages && (
             <>
-              <strong>{username} </strong>
-              {caption}
+              <strong style={{ color: "royalblue" }}>#{username} </strong>
```

Comment:
Also Is # needed?

Reply:
i think it is ok if you want i can remove it.

Reply:
fine